README.md

```diff
@@ -2,14 +2,183 @@
 [![codecov][codecov-badge]][codecov] &nbsp;
 [![license][license-badge]][license]
 
-# Spine Validation SDK
+# Spine Validation
 
-This repository contains Spine Validations library and development tools for working with 
-data types validation.
+The library brings data validation directly into your Protobuf messages.
 
-## Java Support
+### Why Use
 
-This library is built with Java 11.
+Protobuf messages are defined using a language that lacks built-in mechanisms for enforcing 
+domain-specific rules at runtime. As a result, developers must manually insert validation logic
```

Comment:
"must" is too strong a word.

I would first say why we need validation as an idea. And then say how they would do it without this library.